doc/api.yml

```diff
@@ -1738,6 +1762,11 @@ components:
               format: date-time
             status:
               $ref: '#/components/schemas/EnableStatus'
+    Sex:
+      type: string
+      enum:
+        - M
+        - F
```

Comment:
refactor your User schema to use this component

Reply:
checked